docs/tutorials/kubernetes-basics/expose-intro.html

```diff
@@ -31,7 +31,7 @@
 
 			<p>This abstraction will allow us to expose Pods to traffic originating from outside the cluster. Services have their own unique cluster-private IP address and expose a port to receive traffic. If you choose to expose the service outside the cluster, the options are:</p>
 			<ul>
-				<li>LoadBalancer - provides a public IP address (what you would typically use when you run Kubernetes on GKE or AWS)</li>
```

Comment:
GCE should probably be "GCP". Google Cloud Platform is the equivalent to Amazon Web Services (from a descriptive standpoint).